docs/podman-restart.1.md

```diff
@@ -12,33 +12,51 @@ Containers will be stopped if they are running and then restarted. Stopped
 containers will not be stopped and will only be started.
 
 ## OPTIONS
-**--timeout**
-
-Timeout to wait before forcibly stopping the container
+**--all, -a**
+Restart all containers regardless of their current state.
 
 **--latest, -l**
-
 Instead of providing the container name or ID, use the last created container. If you use methods other than Podman
 to run containers such as CRI-O, the last started container could be from either of those methods.
 
+**--running-only**
+Restart all containers that are already in the *running* state.
+
+**--timeout**
+Timeout to wait before forcibly stopping the container
```

Comment:
missing period

Reply:
I'm not sure, if --timeout can be used with --all or --running, should we change to "stopping a container" as there might be multiples and I'm assuming the timeout would apply individually?

Reply:
OK last pick, "Timeout in seconds"

Reply:
fixed